chart/templates/core-agents-deployment.yaml

```diff
@@ -39,4 +39,14 @@ spec:
             - name: MY_POD_NAMESPACE
               valueFrom:
                 fieldRef:
-                  fieldPath: metadata.namespace
\ No newline at end of file
+                  fieldPath: metadata.namespace
+            - name: LLVM_PROFILE_FILE
```

Comment:
It's ignored but it means we're always creating folders in the host systems because of the hostPath, and in that past that has caused issues so IMHO we shouldn't add that to the yaml's by default.

Reply:
This should be part of the "thing" that does the coverage.

Reply:
Fixed.
https://github.com/mayadata-io/mayastor-control-plane/pull/158/commits/c8df674795035432102a90e95b891f9627908aaa